### PR TITLE
14-test-af-simplescorer

### LINKED ISSUE #14: Test af simpleScorer
+ renaming

### PR DESCRIPTION
**Pull request by Uewotm90**
- [x] yderligere test af med nogle edge cases
- [x] snake_case -> camelCase
- [x] tests er blevet flyttet i `describe`'s

### CODE REVIEW

File: src/implementations/SimpleScorer.ts [hunk 1]
```diff
@@ -178,35 +178,34 @@ export function haversine_dist(point_test: [number, number], point_real: [number
...
-  let nice_cog = zip(computed_bearings, messages)
+  let niceCog = zip(computed_bearings, messages)
     .map((x) => [x[0], x[1].cog])
     .filter((x): x is [number, number] => x[1] !== undefined || x !== null || !Number.isNaN(x[1]))
     .map((x) => Math.abs(x[0] - x[1]))
     .map((x) => Math.max(x - TOLERANCE, 0))
     .map((x) => 1 - x / 360)
-
-  return nice_cog
+  return niceCog
```
Comment (JonasGLund99) [inline src/implementations/SimpleScorer.ts:207]: 🍆 🐓

File: src/__tests__/VesselScore.test.ts [hunk 2]
```diff
@@ -109,159 +109,240 @@ function test_mes(): AISJobData {
...
-  new SimpleScorer().score(jobdata).then((res) => {
-    let reason = res.reason!
-    // console.log(score)
-    expect(reason).toBeDefined
-    expect(reason).toBe('')
+    const score = new SimpleScorer().cog_analysis(mes)
+
+    expect(score).toBeNaN()
+  })
+  test('cog inspection', () => {
```
Comment (JonasGLund99) [inline src/__tests__/VesselScore.test.ts:283]: 👓 🍆

**Review by JonasGLund99 — APPROVED** [review]

**Review by Murmeldyret — APPROVED** [review]
iLoveTheUseOfcamel_caseInTheseFiles
